### ISSUE TITLE
Improved Tag Setting Navigation

### ISSUE THREAD

**Issue by TheMan6900** [EmeraldLoc/tag#453]
D-pad (or joystick) to scroll up and down,
A to select settings,
B to go back a category,

These would display near or top of the settings panel for about 5 seconds, after they would completely fade out.

**Comment by EmeraldLoc** [comment]
The only thing I’d change here is the b to go back, everything else is plain worse or already exists

**Comment by TheMan6900** [comment]
Whats wrong with having to press A to select settings

**Comment by EmeraldLoc** [comment]
Your already hovering over the setting, a is just an extra button press.

**Comment by TheMan6900** [comment]
So? Other Ui elements like Voting use A as a selector, why not the regular settings?

**Comment by TheMan6900** [comment]
If anything Dpad right/left to select is less intuitive, than just... pressing a to select

**Comment by EmeraldLoc** [comment]
Oh, you mean hit A to change a setting. I don’t like this, since there’s also a LOT of settings that require that form of movement. Claiming it’s “less intuitive” doesn’t really make sense to me, since mods like mh use that bind, and if up and down does something, left and right must too, it’s all mainly sucbjective though. My opinion is no on that one. Also my this is a long message for a single button.

**Comment by TheMan6900** [comment]
Yeah, aint wanting to start an argument over the button you press to toggle something in a video game menu

**Comment by TheMan6900** [comment]
You should probably label this issue as an enhancement
